go/libkb/id_table.go

```diff
@@ -618,9 +618,13 @@ func (s *EldestChainLink) Type() string                  { return string(Delegat
 func (s *EldestChainLink) ToDisplayString() string       { return s.kid.String() }
 func (s *EldestChainLink) GetDevice() *Device            { return s.device }
 func (s *EldestChainLink) GetPGPFullHash() string        { return s.extractPGPFullHash("key") }
+func (s *EldestChainLink) insertIntoTable(tab *IdentityTable) {
```

Comment:
@oconnor663 this broke `keybase sigs list`, fixed with a specific implementation that wasn't a generic.